docs/source/algorithms/multi_label.rst

```diff
@@ -0,0 +1,231 @@
+
+==========================
+Multi-Path Classification
```

Comment:
```suggestion
Multi-Label Classification
```
I would suggest to stick to multi-label classification in order not to confuse the reader. Multiple instances would need to be updated in this file.

Reply:
Agreed, updated it to _Hierarchical Multi-label_ everywhere.